internal/benchrunner/reporters/reportable.go

```diff
@@ -0,0 +1,84 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package reporters
+
+// Reportable defines a raw report associated to a package.
+type Reportable interface {
+	Package() string
+	Report() []byte
+}
+
+var _ Reportable = &Report{}
+
+type Report struct {
+	pkg string
+	p   []byte
```

Comment:
what's `p` for?
I would expect something more like `r` (report) or `c` (content) :)